### PR TITLE
Update github issues templates

### PR DESCRIPTION
**Pull request by mathiasvr**
Remove old github issue template and add new ones based on those in the main betaflight repo.

Mostly identical to a PR opened it the Betaflight Configurator repo: https://github.com/betaflight/betaflight-configurator/pull/2665.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (ctzsnooze) [inline .github/ISSUE_TEMPLATE/bug-report.md:22]: This line seems relevant to a flight controller or firmware issue.  I wonder if here, for logs it might be more useful to ask the user to post the log header, or a link to the log  file itself with which they are having a problem?

Comment (mathiasvr) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Yes that makes sense, I updated the PR to specify a log instead.

**Review by ctzsnooze — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
